### ISSUE TITLE
Create a security policy

### ISSUE THREAD

**Issue by JamieSlome** [Mailu/Mailu#1872]
Hi there,

I couldn't find a `SECURITY.md` in your repository and am not sure how to best contact you privately to disclose a security issue.

Can you add a `SECURITY.md` file with an e-mail to your repository, so that our system can send you the vulnerability details? GitHub suggests that a [security policy](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) is the best way to make sure security issues are responsibly disclosed.

Once you've done that, you should receive an e-mail within the next hour with more info.

Thanks! (cc @huntr-helper)

**Comment by Diman0** [comment]
Thank you for reaching out to us. We indeed have no security policy and no procedure for this. 
We will create a security.md file with details via what email address you can send the report. This might take a couple of days.

**Comment by Diman0** [comment]
@nextgens I arranged we got an email address for the mailu domain that could be used. It is security at mailu.io.  I can configure email forwarding for this email address. It is a matter of creating the policy and after that I will enable forwarding for anyone who want to receive such emails.

There is no rush. This is the first time someone asked for this policy. I do not think it is likely someone else will come with this question in the near future.

For anyone reading this, at this moment no email forwarding is set up for this email address. Don't bother sending emails, the mail box is not **monitored** at this moment.
